client/src/containers/pages/Home/Home.tsx

```diff
@@ -43,6 +45,15 @@ const Home: React.FC<HomeProps> = (props: HomeProps) => {
             </Link>
           </Col>
         )}
+        {ENABLE_TEAMS && (
```

Comment:
Hope we've also added a test case for this scenario too just like we've added tests for enabled menu links in the sidebar [here](https://github.com/OpenSRP/web/blob/master/client/src/containers/ConnectedSidebar/tests/index.test.tsx#L51)